src/main/web/templates/handlebars/partials/interactive.handlebars

```diff
@@ -3,7 +3,7 @@
     <div class="pym-interactive{{#if_eq full-width "true"}} pym-interactive--full-width margin-left--0{{/if_eq}}" id="{{id}}" data-url="{{url}}"></div>
 
     <details class="details">
-        <summary class="details__summary">Embed code</summary>
+        <summary class="details__summary" aria-expanded="false">Embed code</summary>
```

Comment:
Which browser is it not announcing `details` controls?

Reply:
The issue was raised against Voiceover on iPhone.
I've had it also not be announced by ChromeVox in Chrome.